src/callAPIMethod.js

```diff
@@ -20,21 +20,29 @@ export default function callAPI(
     methodOptions = {}
 ) {
 
-    const { path=[], query={}, options={}, method='json' } = methodOptions;
+    const { path=[], query={}, options={}, method='json', headers, body } = methodOptions;
```

Comment:
Good point. 
Yeah i needed a way to easily send post request and align a bit more with how axios does things